CODEOWNERS

```diff
@@ -95,6 +95,10 @@
 /stepactions/eaas-get-supported-ephemeral-cluster-versions  @amisstea @avi-biton @hmariset @omeramsc @yftacherzog
 /task/eaas-provision-space                                  @amisstea @avi-biton @hmariset @omeramsc @yftacherzog
 
+# renovate groupName=sealights
+/task/sealights-go         @ascerra @flacatus @psturc @dbaez
+/task/sealights-python     @ascerra @flacatus @psturc @dbaez
```

Comment:
Invited you and @ascerra to https://github.com/orgs/konflux-ci/teams/build-definition-contributors.

Is `dbaez` the correct username? https://github.com/dbaez